packages/common/core-interfaces/src/messages.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @internal
+ */
+export interface ISignalEnvelope {
```

Comment:
How is this different from ISignalEnvelope in pacakges/runtime/runtime-definitions/src/protocol.ts?
Should we have only one copy of that? Does it belong to protocol packages (maybe we stage changes, but we need to answer on where it should belong at the end).

Reply:
It is same. I deprecated the other one because it could be used anywhere. So deprecated in runtime definitions. If the server does not use it, then I thought we should just keep it in client packages. It will also lead to easier development.